README.md

```diff
@@ -296,19 +199,20 @@ async def print_models() -> None:
 asyncio.run(print_models())
 ```
 
-## Elevenlabs module
-All of the ElevenLabs models are nested within the elevenlabs module. 
+## Elevenlabs Namespace
+
+All of the ElevenLabs models are nested within the elevenlabs module.
 
-![Alt text](assets/module.png)
+<img src="https://github.com/elevenlabs/elevenlabs-js/blob/main/assets/namespace.png" width="900">
```

Comment:
I wouldn't link to assets in another repo, we won't know we are breaking this when changing/deleting the image in the other repo.

And honestly I'm not sure what the picture accomplishes, can we just get rid of it?